src/main/java/life/qbic/io/commandline/PostmanCommandLineOptions.java

```diff
@@ -61,7 +67,13 @@ void download(
             authentication.getSessionToken(),
             outputPath);
     ids = verifyProvidedIdentifiers();
-    qbicDataDownloader.downloadRequestedFilesOfDatasets(ids, suffixes);
+    DownloadResponse downloadResponse = qbicDataDownloader.downloadForIds(ids, suffixes);
+    LOG.info("Done");
+    if (downloadResponse.containsFailure()) {
+      LOG.warn(String.format("Failed to download %s out of %s files",
+          downloadResponse.failureCount(), downloadResponse.fileCount()));
+      System.exit(1);
```

Comment:
Alternatively you can introduce a static class with constants, for example `APPLICATION.EXIT_CODE.FAILURE` and `APPLICATION.EXIT_CODE.SUCCESS`

Reply:
I don't think we should introduce more exit codes than `0` and `1`. If this is needed in the future I would vote to extend it then. Right now I prefer staying with the default `0` and `1` codes.